packages/css/test/index.js

```diff
@@ -77,6 +86,23 @@ test('returns system props styles', () => {
   })
 })
 
+test('[theme:object breakpoints] returns system props styles', () => {
```

Comment:
What is this array-like convention in the test description?

Reply:
I just copied the tests and used the theme with object breakpoints. Don't really know what the best approach is to describe the test.

Reply:
I think for simplicity sake, you can augment the previous test ("returns system props styles"), and add an extra CSS property using object styles.